<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BarCodePage</class>
 <widget class="QWidget" name="BarCodePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>471</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>输入条形码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="codeInput"/>
    </item>
    <item>
     <widget class="QPushButton" name="CreateBarCodeButton">
      <property name="text">
       <string>生成</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="BarCodeShow">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>461</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
